### PR TITLE
Fb issue 555

### LINKED ISSUE #555: Remove all references/code to PersistenceManager
There is still an old 'caching' construct in place around the PersistenceManager interface:
Package com.dexels.navajo.persistence in com.dexels.navajo.document
Package com.dexels.navajo.persistence.impl in com.dexels.navajo.core
This construction is used anymore except in some test classes (check this).

### PR DESCRIPTION
**Pull request by aschoneveld**
This change involves removing all references in old piece of code in Navajo: PersistenceManager/Constructable/etc.

### CODE REVIEW

**Comment by aschoneveld** [comment]
@gballintijn : when approved by all reviewers can you merge this in master?

**Comment by gballintijn** [comment]
@aschoneveld Ok.

**Review by billgemist — APPROVED** [review]

File: core/com.dexels.navajo.core/src/com/dexels/navajo/server/Dispatcher.java [hunk 1]
```diff
@@ -350,13 +350,6 @@ private final Navajo dispatch(Access access) throws Exception {
...
-            // If recompile is needed ALWAYS set expirationInterval to -1.
-            // ALSO I DO NOT WANT CACHECONTROLLER DEPENDENCY @ THIS POINT.
-            long expirationInterval = CacheController.getInstance().getExpirationInterval(access.rpcName);
-            if (expirationInterval > 0 && sh.needsRecompile( access )) {
-                expirationInterval = -1;
-            }
-
             // Remove password from in to create password independent
             // persistenceKey.
             in.getHeader().setRPCPassword("");
```
Comment (gballintijn) [inline core/com.dexels.navajo.core/src/com/dexels/navajo/server/Dispatcher.java:355]: @aschoneveld  Is this line still needed; the comment above and the rest of your changes suggest it is no longer needed; or am I missing something?

**Review by mbergman — APPROVED** [review]

File: core/com.dexels.navajo.core/src/com/dexels/navajo/server/Dispatcher.java (hunk 1 above)
Comment (aschoneveld) [reply core/com.dexels.navajo.core/src/com/dexels/navajo/server/Dispatcher.java:355]: Yes, that line can be removed. I will do that.

**Review by ghost — APPROVED** [review]

**Pull request merged**